AppCenter/AppCenter/Internals/Model/Util/MSUserDefaults.h

```diff
@@ -15,6 +15,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 + (instancetype)shared;
 
++ (void)resetSharedInstance;
+
+/**
+ * Migrates the given array of keys: re-saves the value by the old key to new key.
+ * Deletes the old key.
+ * @param migratedKeys keys to be migrated.
+ * Dictionary format: [oldKey : newKey].
+ * @param serviceName name of the service executing the migration.
```

Comment:
Not sure that it's correct format

Reply:
@MatkovIvan what place exactly?

Reply:
I think he's referring to line 24.

Reply:
Still not sure what's wrong. This is how I pass the keys to the method

Reply:
Something like that:
```suggestion
 * Deletes the old key.
 *
 * @param migratedKeys keys to be migrated. Dictionary format: [oldKey : newKey].
 * @param serviceName name of the service executing the migration.
```